libvast/src/plugin.cpp

```diff
@@ -336,6 +353,19 @@ const std::vector<std::filesystem::path>& loaded_config_files() {
 
 } // namespace plugins
 
+// -- plugin base class -------------------------------------------------------
+
+bool plugin::enabled(const record&, const record& plugin_config) const {
+  auto default_value = true;
+  return get_or(plugin_config, "enabled", default_value);
```

Comment:
This should probably be `try_get_or`, so that you get an error if the value is set to something other than a boolean.